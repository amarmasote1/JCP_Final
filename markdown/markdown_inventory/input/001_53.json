
{
	"key": "JCP|1398|70630003003|EACH"
}  