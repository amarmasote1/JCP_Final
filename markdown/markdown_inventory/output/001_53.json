{
    "warnings": null,
    "message": "Success",
    "body": null,
    "key": "JCP|1398|70630003003|EACH",
    "status": 200
}