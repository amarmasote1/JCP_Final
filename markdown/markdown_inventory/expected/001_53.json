{"status":200,"message":"Success","key":"JCP|1398|70630003003|EACH","body":null,"warnings":null}